### PR TITLE
Obnb 752

### PR DESCRIPTION
**Pull request by RanjithRajashekarappa**

### CODE REVIEW

File: dnb_developer.json [hunk 1]
```diff
@@ -544,7 +544,7 @@
           "in" : "body",
           "required" : true,
           "schema" : {
-            "$ref" : "#/definitions/paymentInitiation"
+            "$ref" : "#/definitions/paymentUpdation"
```
Comment (DidierParat) [inline dnb_developer.json]: Maybe "updatePayment" instead?

File: dnb_developer.json [hunk 2]
```diff
@@ -544,7 +544,7 @@
           "in" : "body",
           "required" : true,
           "schema" : {
-            "$ref" : "#/definitions/paymentInitiation"
```
Comment (DidierParat) [inline dnb_developer.json]: Please remove the code corresponding to "paymentInitiation".

File: dnb_developer.json (hunk 1 above)
Comment (DidierParat) [inline dnb_developer.json]: Maybe "updatePayment" instead?

Comment (DidierParat) [inline dnb_developer.json]: Maybe "updatePayment" instead?

Comment (DidierParat) [inline dnb_developer.json]: Maybe "updatePayment" instead?

Comment (DidierParat) [inline dnb_developer.json]: Maybe "updatePayment" instead?

Comment (DidierParat) [inline dnb_developer.json]: Maybe "updatePayment" instead?

Comment (DidierParat) [inline dnb_developer.json]: Maybe "updatePayment" instead?

Comment (DidierParat) [inline dnb_developer.json]: Maybe "updatePayment" instead?

Comment (RanjithRajashekarappa) [reply dnb_developer.json]: Renamed to updatePayment

File: dnb_developer.json (hunk 2 above)
Comment (RanjithRajashekarappa) [reply dnb_developer.json]: the code corresponding to paymentInitiation cannot be removed as its a body definition for initaite payment API.

**Comment by relekang** [comment]
Please update the title to be more descriptive, when viewing the merged PR list we should be able to find PRs without cross checking jira.

**Pull request merged**